<?xml version="1.0" encoding="UTF-8"?>
<!-- Path: gui\arquivos_deletados\arquivos_deletados_dialog.ui -->
<ui version="4.0">
 <class>ArquivosDeletedDialog</class>
 <widget class="QDialog" name="ArquivosDeletedDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arquivos Deletados</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="filesTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="firstPageButton">
       <property name="text">
        <string>Primeira</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="prevPageButton">
       <property name="text">
        <string>Anterior</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pageInfoLabel">
       <property name="text">
        <string>Página 1 de 1 (Total: 0 itens)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextPageButton">
       <property name="text">
        <string>Próxima</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lastPageButton">
       <property name="text">
        <string>Última</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Itens por página:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="pageSizeComboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string>Atualizar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportCSVButton">
       <property name="text">
        <string>Exportar CSV</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Fechar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>